package.json

```diff
@@ -3,7 +3,7 @@
   "displayName": "Vim",
   "description": "Vim emulation for Visual Studio Code",
   "icon": "images/icon.png",
-  "version": "0.0.8",
+  "version": "0.0.9",
```

Comment:
Can you revert this change? I'd like to rev the version on a difference schedule (ie. grouping several changes into one version).